CHANGES.txt

```diff
@@ -28,6 +28,9 @@ Breaking Changes
 Changes
 =======
 
+- Fixed an issue that could cause ``IngestRulesListeners`` to forget about the 
```

Comment:
I think this needs some translation for end-users.

How could this happen to a user and what was the observable result from a end-user perspective?

Reply:
"and consequently stop ingesting" - do you think this needs more elaboration? 

In terms of "how could this happen", I don't think this is the place to elaborate on how we rely on cluster events and such or maybe you have something else on your mind?

Reply:
My point is that for users `IngestRuleListeners` is probably as meaningful as `Banana`. The release notes should be writter with end-users in mind and we need to translate internals so that it's understandable for someone who has no clue about any internals.
Maybe something along the lines of:

> Internal cluster state updates could reset the active ingestion rules, causing all ingest operations to stop.

If a specific user action could trigger this, it might make sense to point that out as well.